client/src/test/java/com/influxdb/client/ITInfluxQLQueryApi.java

```diff
@@ -89,7 +107,8 @@ void testQueryData() {
 				.hasSize(1)
 				.first()
 				.satisfies(record -> {
-					Assertions.assertThat(record.getValueByKey("time")).isEqualTo("1655900000000000000");
+//					Assertions.assertThat(record.getValueByKey("time")).isEqualTo("1655900000000000000");
+					Assertions.assertThat(record.getValueByKey("time")).isEqualTo("2022-06-22T12:13:20Z");
```

Comment:
Is this caused by a breaking change?

Reply:
Yes.  because `Accept: application/csv` returns an Epoch timestamp (which was covered by the original test version) and `Accept: application/json` returns the timestamp as an RFC3339 string.